Missing dependencies after installation with npm@5.2

After installation of `tslint-react` with npm v5.2.0 I got a
```
Warning: Cannot find module 'tsutils' Use --force to continue.
```
After digging through the sources I found that rules implementations (i.e. `tslint-react/rules/jsxAlignmentRule.js`) require node modules `tslint`, `typescript` and `tslint` but not any of these modules is listed in `dependencies` or `peerDependencies`.

--- a/README.md
+++ b/README.md
@@ -9,8 +9,7 @@ Lint rules related to React & JSX for [TSLint](https://github.com/palantir/tslin
 
 ### Usage
 
-tslint-react has _implicit_ peer dependencies on TSLint and TypeScript. The recommended versions of those packages are
-listed as `devDependencies`.
+tslint-react has peer dependencies on TSLint and TypeScript.
 
 To use these lint rules with the default preset, use configuration inheritance via the `extends` keyword.
 Here's a sample configuration where `tslint.json` lives adjacent to your `node_modules` folder:

--- a/package.json
+++ b/package.json
@@ -18,6 +18,11 @@
     "test-rules": "scripts/testRules.sh",
     "verify": "npm-run-all build test-rules"
   },
+  "peerDependencies": {
+    "tslint": "^5.1.0",
+    "tsutils": "^1.7.0",
+    "typescript": ">=2.1.0 || >=2.1.0-dev || >=2.2.0-dev || >=2.3.0-dev || >=2.4.0-dev || >=2.5.0-dev || >=2.6.0-dev"
+  },
   "devDependencies": {
     "@types/colors": "^1.1.3",
     "@types/glob": "^5.0.30",

